metals/src/main/scala/scala/meta/metals/MetalsServices.scala

```diff
@@ -42,12 +42,12 @@ import monix.reactive.Observable
 import monix.reactive.Observer
 import monix.reactive.OverflowStrategy
 import org.langmeta.inputs.Input
-import org.langmeta.internal.semanticdb.XtensionDatabase
-import org.langmeta.internal.semanticdb.schema
+import scala.meta.internal.semanticdb3
```

Comment:
curiosity: why are `semanticdb3` data types internal?

Reply:
Because otherwise we'd have to provide strong compatibility guarantees for them: https://github.com/scalameta/scalameta/blob/master/COMPATIBILITY.md. I don't think we're fully ready to do this right now.